stdlib/hashtbl.ml

```diff
@@ -377,21 +406,32 @@ module type S =
     val create: int -> 'a t
     val clear : 'a t -> unit
     val reset : 'a t -> unit
-    val copy: 'a t -> 'a t
-    val add: 'a t -> key -> 'a -> unit
-    val remove: 'a t -> key -> unit
-    val find: 'a t -> key -> 'a
+    val copy : 'a t -> 'a t
```

Comment:
No reason for these whitespace changes (and the recommended style would rather be the one without the whitespace).